EIPS/eip-1.md

```diff
@@ -42,19 +42,23 @@ The EIP process begins with a new idea for Ethereum. It is highly recommended th
 
 Each EIP must have a champion - someone who writes the EIP using the style and format described below, shepherds the discussions in the appropriate forums, and attempts to build community consensus around the idea.
 
-Vetting an idea publicly before going as far as writing an EIP is meant to save the potential author time. Asking the Ethereum community first if an idea is original helps prevent too much time being spent on something that is guaranteed to be rejected based on prior discussions (searching the Internet does not always do the trick). It also helps to make sure the idea is applicable to the entire community and not just the author. Just because an idea sounds good to the author does not mean it will work for most people in most areas where Ethereum is used. Examples of appropriate public forums to gauge interest around your EIP include [the Ethereum subreddit], [the Issues section of this repository], and [one of the Ethereum Gitter chat rooms]. In particular, [the Issues section of this repository] is an excellent place to discuss your proposal with the community and start creating more formalized language around your EIP.
+Vetting an idea publicly before going as far as writing an EIP is meant to save the potential author time. Asking the Ethereum community first if an idea is original helps prevent too much time being spent on something that is guaranteed to be rejected based on prior discussions (searching the Internet does not always do the trick). It also helps to make sure the idea is applicable to the entire community and not just the author. Just because an idea sounds good to the author does not mean it will work for most people in most areas where Ethereum is used. Examples of appropriate public forums to gauge interest around your EIP include [the Fellowship of Ethereum Magicians], [the Ethereum subreddit], [the Issues section of this repository], and [one of the Ethereum Gitter chat rooms]. In particular, [the Issues section of this repository] is an excellent place to discuss your proposal with the community and start creating more formalized language around your EIP.
+
```

Comment:
There is long running eip-4 draft which does not match the classification in eip-1. Perhaps it's an issue for eip-4 rather than here but considering that eip-4 has been open since 2015 perhaps close it to avoid confusion around definitions?

Reply:
Do you mean https://github.com/ethereum/EIPs/pull/4? That is closed.

Reply:
Oh you mean https://github.com/ethereum/EIPs/blob/master/EIPS/eip-4.md.

Reply:
The titles are clearly different, except for "Networking"?
"Consensus" vs "Core"
"API/RPC" vs "Interface"
"Applications" vs "ERC"

Reply:
There is a lot of overlap.

Reply:
Any way, there is already a description of the subcategories in EIP-1, so at a glance, it seems there's no need for EIP-4, IMO.

Reply:
Agree, I see that going into the descriptions of each classification, I only said that eip-4 has to be addressed as part of the effort to clean up governance.

Reply:
I agree, EIP 4 should probably be transitioned to obsolete.